spec/release/notes/configuration_spec.rb

```diff
@@ -119,4 +119,90 @@
       end
     end
   end
+
+  describe "#release_notes_exist?" do
+    context "when output file does exist" do
+      it "returns true" do
+        expect(Release::Notes.configuration.release_notes_exist?).to eq false
+      end
+    end
+    context "when output file does not exist" do
+      it "returns false" do
+        if Release::Notes.configuration.release_notes_exist?
```

Comment:
Maybe just an overlooked mistake - we wouldn't want a conditional in our test here. Let's stub this or setup our test so that it's always true.

Reply:
👍